pallets/funding/src/tests/2_evaluation.rs

```diff
@@ -618,6 +624,116 @@ mod evaluate_extrinsic {
 				assert_eq!(stored_evaluation, &expected_evaluation_item);
 			});
 		}
+
+		#[test]
+		fn can_evaluate_with_frozen_tokens() {
+			let mut inst = MockInstantiator::new(Some(RefCell::new(new_test_ext())));
+			let issuer = ISSUER_1;
+			let project_metadata = default_project_metadata(issuer);
+			let project_id = inst.create_evaluating_project(project_metadata.clone(), issuer);
+
+			let evaluation = UserToUSDBalance::new(EVALUATOR_4, 500 * USD_UNIT);
+			let plmc_required = inst.calculate_evaluation_plmc_spent(vec![evaluation.clone()]);
+			let frozen_amount = plmc_required[0].plmc_amount;
+			let plmc_existential_deposits = plmc_required.accounts().existential_deposits();
+
+			inst.mint_plmc_to(plmc_existential_deposits);
+			inst.mint_plmc_to(plmc_required.clone());
+
+			inst.execute(|| {
+				mock::Balances::set_freeze(&(), &EVALUATOR_4, plmc_required[0].plmc_amount).unwrap();
+			});
+
+			inst.execute(|| {
+				assert_noop!(
+					Balances::transfer_allow_death(RuntimeOrigin::signed(EVALUATOR_4), ISSUER_1, frozen_amount,),
+					TokenError::Frozen
+				);
+			});
+
+			inst.execute(|| {
+				assert_ok!(PolimecFunding::evaluate(
+					RuntimeOrigin::signed(EVALUATOR_4),
+					get_mock_jwt_with_cid(
+						EVALUATOR_4,
+						InvestorType::Retail,
+						generate_did_from_account(EVALUATOR_4),
+						project_metadata.clone().policy_ipfs_cid.unwrap()
+					),
+					project_id,
+					evaluation.usd_amount
+				));
+			});
+
+			let new_evaluations = default_evaluations();
```

Comment:
Why do we need more default evaluations? Couldn't we just fill the required evaluation with Evaluator_4? And couldn't we just create a finished_project with mostly default values (or empty for bids / contributions as we are doing here)? Or does that error if we do not have the required bids / contributions?

Reply:
The default evaluations was unnecessary, nice catch.
We can't use create_finished_project because we have to set a freeze before evaluating, and the account must exist before I evaluate.

If I create the account before calling create_finished_project, then I will have 2 EDs on the account, and will make the testing values weird. So prefer to keep as is